### DISCUSSION TITLE
What are best practices for archival backups?

### DISCUSSION THREAD

**Discussion by riedel** [FraunhoferIOSB/FROST-Server#2078] [Q&A]
Hi,

we have an old frost server running with a few million observations, which we want to archive. 

My question what is the best option for archival backups?

SQL dumps would be rather relying on the actual FROST server version (before the JSONB migration, which failed, why we sticked on the old version) rather than actually only the FROST standard.

Is there an export/import function that does not rely on the actual backend? I can imagine exporting it, but it would have no value if we cannot import again.

Best

Till

PS: I would be fine with a few import errors (we have null strings that we need to better escape for newer FROST versions).

**Comment by hylkevds** [comment]
Currently an SQL dump is the only way to make a backup. If it's from a really old version it may be easier/faster to copy the data manually over using SQL commands.

I've been thinking about making a plugin that can export all data in JSON-Batch format, but have not had resources to look further into it.
